sys-devel/clang-common/clang-common-17.0.6-r5.ebuild

```diff
@@ -68,11 +68,17 @@ _doclang_cfg() {
 
 	local tool
 	for tool in ${triple}-clang{,++}; do
-		newins - "${tool}.cfg" <<-EOF
+		cat > "${T}/${tool}.cfg" <<-EOF || die
```

Comment:
Why the extra indirection? Just append to the installed file, like the ebuild does elsewhere.

Reply:
Mostly personal preference: I dislike modifying files once they have been installed to `${D}`. It feels cleaner to build the file elsewhere (preferably in a userpriv-protected phase) and then copy it once with doins. Also, I dislike referring to the same file with 2 different paths ("${tool}.cfg" and "${ED}/etc/clang/${tool}.cfg").

However, this is not my ebuild, so I have adjusted it as requested.